api/server/objectstore.go

```diff
@@ -0,0 +1,140 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/libopenstorage/openstorage/objectstore"
+)
+
+// swagger:operation GET /cluster/objectstore objectstore objectStoreInspect
+//
+// Lists Objectstore
+//
+// This will list current object stores
+//
+// ---
+// produces:
+// - application/json
+// responses:
+//   '200':
+//     description: success
+//     schema:
+//      $ref: '#/definitions/ObjectstoreInfo'
+func (c *clusterApi) objectStoreInspect(w http.ResponseWriter, r *http.Request) {
+	method := "objectStoreInspect"
+
+	objInfo, err := c.ObjectStoreManager.ObjectStoreInspect()
```

Comment:
Doesn't Inspect need to take the volume ID just as Create() did?

Reply:
No it doesn't. There is only one objectstore can be running on PX cluster with one volume as backend, I guess that's reason Inspect does not take volumeID. Same for Update and Delete.

@disrani-px  Please let me know If I'm missing something here

Reply:
This is a limitation of the implementation of Px, but not of OpenStorage. The interface should not assume this limitation. There should be a setting in the VolumeSpec also showing that this volume has enabled ObjectStorage with a link to the objectstorage info object.

Reply:
Yes, right now only one objectstore can be created in PX.

Reply:
I disagree, the volume should not know what it is being used for.

Reply:
The Create API can always return the same object as Inspect API.

Reply:
There is no mapping from volumeID to objectstore. In the future, there might be a volume for each bucket, so that mapping doesn't make sense. You can pass in the objectstore ID to update/delete/inspect.

Reply:
sounds good